### DISCUSSION TITLE
[PvP] Reverting Arena Point increase from Daily Battleground Win and implementing Arena Points per win

### DISCUSSION THREAD

**Discussion by Belodina** [chromiecraft/chromiecraft#7429] [Suggestions]
This suggestion aims to address the meta of logging in for one battleground and not engaging further with the battleground system. While increasing Arena Point (AP) rewards for players winning battlegrounds to stimulate PvP engagement is arguably necessary (whether AP should come exclusively from Arena is a different discussion, Erson), the manner in which these rewards are delivered is important. With the current daily-win bonus system, players are incentivised to only play until they win their first battleground. With the proposed change, players are rewarded per battleground won, incentivising continued engagement in battlegrounds.

### Proposed Values

The current first battleground win of the day bonus gives 20 AP. The suggested values are:

10 Arena Points for the first battleground win (reverted from 20)
5 Arena Points for every subsequent battleground win (new)

The daily first Call-to-Arms (CtA) Win Bonus will remain a flat 25 AP

### Expected Results

This change should result in players queueing for battlegrounds more than once in an evening. Battlegrounds on non-CtA days should no longer be 10v10 > 8v8 > 6v6 >dedge for example. This also further rewards dedicated players, allowing them to exceed the daily "allowance" of Arena Points from battlegrounds.

_This suggestion is directly taken from the good ideas of PvPers on the ChromieCraft Discord. This suggestion simply acts as a formal suggestion through the official channels._

**Comment by TotallyNotMyAccount** [comment]
Arena points should mostly come from arena. Giving out ever more AP to people who barely do any arena will further damage the already precarcious situation the arena community is in already. To spell this out clearly, I am very opposed to this idea. Find a different carrot to dangle in front of people if you want to stimulate BG attendance.

**Comment by vvranceanu** [comment]
If you want arena gear, play arena. you want to "increase pvp" by decreasing pvp. please stop asking for all kind of kitsch to magically circumvent low pop, you might get taken seriously
